### ISSUE TITLE
Bitimages

### ISSUE THREAD

**Issue by igormages** [fasteroko/StarIOPrint-cordova#3]
Hi! Thanks for sharing this amazing plugin !

Do you think that it's possible to add bitimage printing and cash drawer support ?

Thanks !

**Comment by vkornieiev** [comment]
Yes, it possible, we can add support both printing image from data and from
url (if it needed). But this functional will be added on IOS plugin
firstly, and then a bit later on Android.
Thank you for your feedback, I''m glad to work for people who need this
functionality.
Vladyslav.

2016-10-12 1:41 GMT+03:00 Igor Magès notifications@github.com:
